src/database.ts

```diff
@@ -3536,6 +3643,7 @@ callbackifyAll(Database, {
     'create',
     'batchCreateSessions',
     'batchTransaction',
+    'batchWrite',
```

Comment:
Same Why do we need to add this method here, this method does not takes a callback ?

Reply:
Yeah, it doesn' takes a callback directly

Reply:
Got it. Its an exclude list.